pkg/security/BUILD.bazel

```diff
@@ -100,74 +101,9 @@ go_test(
         "//pkg/util/timeutil",
         "//pkg/util/uuid",
         "@com_github_cockroachdb_errors//:errors",
+        "@com_github_prometheus_client_model//go",
         "@com_github_stretchr_testify//require",
         "@org_golang_x_exp//rand",
-    ] + select({
-        "@io_bazel_rules_go//go/platform:aix": [
-            "//pkg/util/log/eventpb",
-            "@com_github_prometheus_client_model//go",
-            "@org_golang_x_sys//unix",
-        ],
-        "@io_bazel_rules_go//go/platform:android": [
-            "//pkg/util/log/eventpb",
-            "@com_github_prometheus_client_model//go",
-            "@org_golang_x_sys//unix",
-        ],
-        "@io_bazel_rules_go//go/platform:darwin": [
-            "//pkg/util/log/eventpb",
-            "@com_github_prometheus_client_model//go",
-            "@org_golang_x_sys//unix",
-        ],
-        "@io_bazel_rules_go//go/platform:dragonfly": [
-            "//pkg/util/log/eventpb",
-            "@com_github_prometheus_client_model//go",
-            "@org_golang_x_sys//unix",
-        ],
-        "@io_bazel_rules_go//go/platform:freebsd": [
-            "//pkg/util/log/eventpb",
-            "@com_github_prometheus_client_model//go",
-            "@org_golang_x_sys//unix",
-        ],
-        "@io_bazel_rules_go//go/platform:illumos": [
-            "//pkg/util/log/eventpb",
-            "@com_github_prometheus_client_model//go",
-            "@org_golang_x_sys//unix",
-        ],
-        "@io_bazel_rules_go//go/platform:ios": [
-            "//pkg/util/log/eventpb",
-            "@com_github_prometheus_client_model//go",
-            "@org_golang_x_sys//unix",
-        ],
-        "@io_bazel_rules_go//go/platform:js": [
-            "//pkg/util/log/eventpb",
-            "@com_github_prometheus_client_model//go",
-            "@org_golang_x_sys//unix",
-        ],
-        "@io_bazel_rules_go//go/platform:linux": [
-            "//pkg/util/log/eventpb",
-            "@com_github_prometheus_client_model//go",
-            "@org_golang_x_sys//unix",
-        ],
-        "@io_bazel_rules_go//go/platform:netbsd": [
-            "//pkg/util/log/eventpb",
-            "@com_github_prometheus_client_model//go",
-            "@org_golang_x_sys//unix",
-        ],
-        "@io_bazel_rules_go//go/platform:openbsd": [
-            "//pkg/util/log/eventpb",
-            "@com_github_prometheus_client_model//go",
-            "@org_golang_x_sys//unix",
-        ],
-        "@io_bazel_rules_go//go/platform:plan9": [
-            "//pkg/util/log/eventpb",
-            "@com_github_prometheus_client_model//go",
-            "@org_golang_x_sys//unix",
-        ],
-        "@io_bazel_rules_go//go/platform:solaris": [
-            "//pkg/util/log/eventpb",
-            "@com_github_prometheus_client_model//go",
-            "@org_golang_x_sys//unix",
-        ],
-        "//conditions:default": [],
-    }),
+        "@org_golang_x_sys//unix",
+    ],
```

Comment:
@rickystewart any idea why gazelle made this change?

Reply:
Ah, it feels like the old-style tag was preventing it from understanding that we don't build that file on all these platforms.

Reply:
Ah, or it's the other way, it used to be able to exclude the dependency depending on the tag :/

Reply:
I'm guessing our gazelle version predates this change https://github.com/bazelbuild/bazel-gazelle/pull/1243